src/network/graphql/Log.cpp

```diff
@@ -0,0 +1,56 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
```

Comment:
files in graphql/ except grapqhl/gen/ were created manually right ? Why is here microsoft disclaimer ?